br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -602,3 +612,28 @@ func int96ToTime(parquetDate []byte) time.Time {
 	dt := binary.LittleEndian.Uint32(parquetDate[8:])
 	return jdToTime(int32(dt), int64(nano))
 }
+
+func bitStringToByteSlice(v string) ([]byte, error) {
+	ret := make([]byte, 0)
+	// traverse byte reversely
+	for i := len(v); i > 0; i -= 8 {
+		var str string
+		if i-8 < 0 {
+			str = v[0:i]
+			diff := 8 - len(str)
+			for j := 0; j < diff; j++ {
+				// padding with 0
+				str = "0" + str
+			}
+		} else {
+			str = v[i-8 : i]
+		}
+		b, err := strconv.ParseUint(str, 2, 8)
+		if err != nil {
+			return nil, err
+		}
+		// append to the result reversely
```

Comment:
can you use more asymmetric and more various-length data in integration tests to check it's little/big endian